docs/product/components/partials/_topbar.html

```diff
@@ -0,0 +1,151 @@
+                <li>
+                    <a href="#" class="s-topbar--item s-btn s-btn__filled">Log in</a>
+                </li>
+                <li>
+                    <a href="#" class="s-topbar--item s-btn s-btn__primary">Sign up</a>
+                </li>
+            </ol>
+        {% else %}
+            <ol class="s-topbar--content">
+                <li>
+                    <button
+                        class="s-topbar--item s-btn s-btn__icon s-btn__muted d-none sm:d-inline-flex js-search-button"
+                        type="button">
+                            {% icon "Search" %}
+                    </button>
+                </li>
+                {% if isMod %}
+                    <li>
+                        <a href="#" class="s-topbar--item" title="Moderator inbox">
+                            {% icon "Moderator" %}
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="s-topbar--item" title="8 posts flagged for moderator attention">
+                            <span class="s-badge s-badge__bounty">8</span>
+                        </a>
+                    </li>
+                {% endif %}
+                <li>
+                    <a href="#" class="s-topbar--item s-user-card s-user-card__small">
+                        <span class="s-avatar s-avatar__24 s-user-card--avatar">
+                            <img class="s-avatar--image" src="{{ "/assets/img/anonymous-user.svg" | relative_url }}" />
+                            <span class="v-visible-sr">John Doe</span>
+                        </span>
+                        <div class="s-user-card--info">
+                            <ul class="s-user-card--awards">
+                                <li class="s-user-card--rep">3,145</li>
+                                <li class="s-award-bling s-award-bling__gold">3</li>
+                                <li class="s-award-bling s-award-bling__silver">9</li>
+                                <li class="s-award-bling s-award-bling__bronze">20</li>
+                            </ul>
+                        </div>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="s-topbar--item" title="Inbox">
+                        {% icon "Inbox" %}
+                        <span class="s-activity-indicator s-activity-indicator__danger">0</span>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="s-topbar--item" title="Achievements">
+                        {% icon "Achievements" %}
+                        <span class="s-activity-indicator s-activity-indicator__success">+0</span>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="s-topbar--item" title="Review queues">
+                        {% icon "ReviewQueue" %}
+                        <div class="s-activity-indicator">
```

Comment:
I think we want `s-activity-indicator__danger` here.
```suggestion
                        <div class="s-activity-indicator s-activity-indicator__danger">
```

Reply:
Kinda. On production that indicator can be either blue or red, depending on if the queue "is highlighted", which is based on a couple of factors (queue length, any item is "in danger"). Since I already had one red item in the examples, I decided to keep this one blue.

Reply:
I went back and looked at the code again and for the life of me I can't find any trace of the blue (vs red) indicator. Maybe I dreamt it? I'll look over it again, but I might be describing some legacy topbar styles that are no longer in use... I'll update the indicator to be danger to match.